riscv/src/lib.rs

```diff
@@ -330,3 +330,62 @@ fn output_files_from_cargo_build_plan(
 
     assemblies
 }
+
+/// Maps an instruction in .insn syntax to Statement::Instruction() in the expected format.
+///
+/// See https://www.rowleydownload.co.uk/arm/documentation/gnu/as/RISC_002dV_002dFormats.html
+pub fn map_insn_i(
```

Comment:
Nooo! do we really need that?

Reply:
I am open to suggestions. This very low level instruction format is a thing, and we only need it for an inline assembly code in Rust library that uses this to encode a weird `fence` instruction with no arguments on the right side.

Reply:
@chriseth further thoughts?